Gemfile

```diff
@@ -8,10 +8,5 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 ruby "2.3.1"
 
 gem "scraperwiki", github: "openaustralia/scraperwiki-ruby", branch: "morph_defaults"
-gem "nokogiri"
-gem "open-uri-cached"
 gem "pry"
-gem "colorize"
-gem "capybara"
-gem "poltergeist"
-gem 'scraped_page_archive', github: "everypolitician/scraped_page_archive", branch: "master"
+gem 'scraped_page', github: 'everypolitician/scraped_page', branch: 'master'
```

Comment:
Perhaps worth running a rubocop tidy against this file too, so we don't have mismatched quotes like this?